app/templates/records/macros/whats_this_series_or_above_about.html

```diff
@@ -13,7 +13,8 @@ <h2 class="tna-heading-l">What's it about?</h2>
           </div>
           <div
             class="tna-column tna-column--flex-1 tna-column--full-small tna-column--full-tiny tna-column--align-centre tna-!--margin-top-xs">
-            <p>{{ record.short_description }}</p>
+            {% set preferred_description_length = 250 %}
```

Comment:
While it can be defined in any place, would suggest to define template constants at the top using uppercase names for readability and clarity.